bpf_test.go

```diff
@@ -266,4 +303,37 @@ func TestModuleLoadELF(t *testing.T) {
 	checkCgroupProgs(t, b)
 	checkSocketFilters(t, b)
 	checkTracepointProgs(t, b)
+
+	// test 2: New Module, reading from file, with custom context
+	mounted, err := bpffs.IsMounted()
+	if err != nil {
+		t.Fatalf("error checking if %q is mounted: %v", bpffs.BPFFSPath, err)
+	}
+	if !mounted {
+		if err := bpffs.Mount(); err != nil {
+			t.Skipf("error mounting bpf fs, skipping test: %v", err)
+		}
+	}
+
+	bc := elf.NewModuleWithCustomCtx(dummyELF, map[string]*elf.CustomContext{
+		"cgroup/sock": &elf.CustomContext{
+			Name:       "testcc1",
+			CgroupPath: "/sys/fs/cgroup/unified/testcc1",
+		},
+		"cgroup/skb": &elf.CustomContext{
+			Name:       "testcc2",
+			CgroupPath: "/sys/fs/cgroup/unified/testcc2",
+		},
+	})
+	if bc == nil {
+		t.Fatal("prog is nil")
+	}
+	if err := bc.Load(nil); err != nil {
+		if !strings.Contains(err.Error(), "invalid argument") {
+			t.Fatal(err)
+		}
+	}
+	defer bc.Close()
```

Comment:
Actually this is ok, `Close()` reverts the effects of `Load()`. `NewModule()` just allocates some maps and returns a `Module` object